app/actions/local/systems.ts

```diff
@@ -68,3 +68,12 @@ export async function storeConfig(serverUrl: string, config: ClientConfig | unde
     }
     return [];
 }
+
+export async function dismissAnnouncement(serverUrl: string, announcementText: string) {
+    try {
+        const {operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+        await operator.handleSystem({systems: [{id: SYSTEM_IDENTIFIERS.LAST_DISMISSED_BANNER, value: announcementText}], prepareRecordsOnly: false});
```

Comment:
announcementText - is a date value ?

Reply:
No. It is a text value. The way the logic works is that you store the last banner you have dismissed by its text content. So in practice, if I put a banner ("Merry Christmas"), you dismiss it, and I don't put any other banner until the next year again saying "Merry Christmas", I will not see that banner, because my local data thinks I already dismissed it.

That being said, it is the original behaviour, and I think is "good enough".

Just in case, @esethna not sure if this behaviour is what you would expect (or if this feature is in your AOR). Probably not the time to make any changes here (the behaviour is coherent with V1) but maybe is something you want to create a ticket for a future iteration.